### PR TITLE
chore: semantic release tweaks

### PR DESCRIPTION
**Pull request by jo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,9 +14,9 @@ before_install:
   - npm i -g npm@^2.0.0
 before_script:
   - npm prune
-  - 'curl -Lo travis_after_all.py https://git.io/vLSON'
 after_success:
-  - python travis_after_all.py
-  - export $(cat .to_export_back)
   - npm run semantic-release
-  - if [[ $BUILD_LEADER = YES ]]; then npm run test:coverage && npm run test:coverage:upload; fi
```
Comment (gr2m) [inline .travis.yml]: shall we keep the `test:coverage` bit in?

Comment (jo) [inline .travis.yml]: sorry, sure!

**Comment by jo** [comment]
that line is back

**Comment by gr2m** [comment]
Looks good now :+1:

**Pull request merged**